### PR TITLE
LA-224: add missing actions to bottom bar and more actions popup

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
refer to this bug [https://github.com/linagora/linshare-mobile-flutter-app/issues/224](https://github.com/linagora/linshare-mobile-flutter-app/issues/224)

Screen Shot 2021-03-04 at 17 33 37Screen Shot 2021-03-04 at 17 33 43

### CODE REVIEW

**Comment by hoangdat** [comment]
Please do it for SharedSpace and SSD and RS also

File: lib/presentation/widget/myspace/my_space_viewmodel.dart [hunk 1]
```diff
@@ -188,10 +192,12 @@ class MySpaceViewModel extends BaseViewModel {
   }
 
   void shareDocuments(List<Document> documents, {ItemSelectionType itemSelectionType = ItemSelectionType.single}) {
-    if (itemSelectionType == ItemSelectionType.single) {
+    if (_appNavigation.canPop()) {
```
Comment (hoangdat) [inline lib/presentation/widget/myspace/my_space_viewmodel.dart:195]: what is the purpose of it?

**Review by hoangdat — APPROVED** [review]

File: lib/presentation/widget/myspace/my_space_viewmodel.dart (hunk 1 above)
Comment (manhduc-nguyen) [reply lib/presentation/widget/myspace/my_space_viewmodel.dart:195]: I want to make sure that we only pop out the widget if its possible, and I think moving this directly inside the `void popBack()` function is better, shorter code to write. let me change this :)

**Comment by manhduc-nguyen** [comment]
@imGok @hoangdat done, also add missing actions for `SharedSpace too`

**Review by imGok — APPROVED** [review]

**Pull request merged**